### ISSUE TITLE
[Bug] Missing hyperlink in Help Modal

### ISSUE THREAD

**Issue by Xaid-vfx** [creativecommons/chooser#481]
## Description
There is an unnecessary Hyperlink in FAQ's Modal (What should I consider?). The href of this hyperlink is none and on clicking it just reopens the same website in a new Tab.

## Reproduction

1. Go to https://chooser-beta.creativecommons.org/
2. Scroll down to  **CONFUSED? NEED HELP?** 
3. Click on **What should I consider?**
4. Click on **other considerations**

## Expectation

Ideal solution would be to either remove the hyperlink or provide the respective link for other considerations

## Screenshots

<img width="1051" alt="Screenshot 2023-04-04 at 12 28 32 AM" src="https://user-images.githubusercontent.com/77684943/229602020-6e932fdd-38fc-4b6d-bf94-2ffdf41bc513.png">

## Resolution

- [x] I would be interested in resolving this bug.

**Comment by Pursottam6003** [comment]
Hii @Xaid-vfx  I think its a good idea to remove the ``href`` tag  or provide the respective links to the respective sites for the creator 

It will be good idea if i could get the chance to write those pages under the supervision of code maintainers 
and add into the sites 

I would be happy to help 😊

**Comment by owais0017** [comment]
assign this to me. I would love to take a look at the issue.

**Comment by possumbilities** [comment]
@Xaid-vfx This is a good spot, and we're grateful you pointed it out!

I spent some time and looked into this, and the hyperlink in question is not so much unecessary as its missing the correct `href`', which should be `https://wiki.creativecommons.org/wiki/Considerations_for_licensors_and_licensees#Considerations_for_licensors`

There was an error #443 where this got missed when the Help Section content got updated.

I've updated the title of the Issue to be more accurate.
